source/docs/blockchain_configuration.md

```diff
@@ -47,7 +53,8 @@ Similar to [configuring Smart Contracts](/docs/contracts_configuration.html), th
 - **rpcHost**: Host the RPC server listens to
 - **rpcPort**: Port the RPC server listens to
 - **rpcCorsDomain**: The CORS domains the node accepts
-  - By using **"auto"`, Embark checks the other configurations to set the CORS domains. This only adds the required domains.
+  - **auto**: When set to true, Embark checks your other configurations to set the CORS domains. This only adds the required domains.
+  - **additionalCors**: Manual list of CORS domains to accept. If `auto` is set to `true`, those will be the only domains added
```

Comment:
Maybe this could read 
```suggestion
  - **additionalCors**: Manual list of CORS domains to accept. If `auto` is set to `true`, any URLs specified here will be applied *in addition to* those automatically added with `auto`.
```